content/doc/book/using/working-with-projects.adoc

```diff
@@ -46,10 +58,22 @@ When a job is renamed, other jobs that refer to the job by name must be updated
61.2. And I already have a job predefined on this controller. Now if you look at this job, you can see it's named job 1. Not very descriptive. Let's go ahead and take a look at the job and let's figure out what it's really doing. So in this case, what we have is just a Hello World type job. Now again, this is a very contrived example, but follow me through this process. So you can think about how you might want to go ahead and name some of your jobs. So let's go back up and let's assume for a moment that I want to go ahead and name this "Hello". Well, go back to Job1 and on the left nav you'll see "Rename". So let's change Job1 to "Hello". Okay, I like that. If I take a look now, I can see Job1 has been renamed "Hello". And I can also see that my URL also is "Hello". Well, "Hello" is fine, but it's not really exactly what I want. Can you guess where I'm going next? Let's make it Hello World. So let's rename Hello to Hello Space World. When we click on Rename, we can see the pipeline as Hello Space World. But if you look up in the URL, I also see a space here. That's not really what I want. In fact, if I click into it again, that could be really hard to work with from CLIs or using Curl or WGET. So for my URL, I really want to put in a hyphen. So let's go ahead and rename our job one more time here. Let's change this to dash. I click on rename and now my pipeline is named hello-world. If I look up in my URL, it's also a hello-world. That's exactly what I wanted, but really, it's still not what I want. Now at Hello World, I'm okay with here, but looking at my dashboard, I see hello-world. Well, really what I want to show, especially when I'm on the dashboard view in this case, is I want to give this a more human-friendly name. Instead of a URL friendly name, I want to have a human friendly name. Let's go into our job and click on configure and let's scroll down to advanced. Now depending on which job that you're using, this next field, which is called display name, may show up in different places. Sometimes it will be exposed. Sometimes it will be under advanced. So in this case, what I want to have is I want to say, "Hello world from the universe." Let's click on Save. We can see here that our pipeline name now is Hello World from the universe, but our project name or our job name is still Hello Dash World. So if we take a look up in the URL, it's still Hello Dash World, but in the breadcrumb, we see the full display name that we just typed in. And in fact, when we go back to the root of our dashboard, we can see Hello World from the universe. If you have any questions or comments, you can reach out to us on Twitter at CloudBeast. If this video was helpful to you, give us a thumbs up. And if you haven't subscribed to CloudBeesTV yet, why not? Take a moment, click on that subscribe button, and then ring that bell, and you'll be notified anytime there's new content available on CloudBeesTV. Thanks for watching, and we will see you in the next video.
```

Comment:
```suggestion
Rename a job in Jenkins.

Here's today's starting point; I have a Jenkins controller version 2.361.2 and I already have a job predefined on this controller.
Now if you look at this job, you can see it's named "job1", not very descriptive.
Let's go ahead and take a look at the job and let's figure out what it's really doing.
So in this case, what we have is just a `Hello World` type job.
Now again, this is a very contrived example, but follow me through this process so you can think about how you might want to go ahead and name some of your jobs.
So let's go back up and let's assume for a moment that I want to go ahead and name this "Hello".
Well, go back to "job1" and on the left nav you'll see *Rename*.
So let's change "job1" to "hello".
Okay, I like that.

If I take a look now, I can see "job1" has been renamed "Hello", and I can also see that my URL also is `hello`.
Well, "hello" is fine, but it's not really _exactly_ what I want.
Can you guess where I'm going next?
Let's make it "hello world".
So let's rename "hello" to "hello world".
When we click on *Rename*, we can see the pipeline as "hello world".
But if you look up in the URL, I also see a space here.
That's not really what I want.
In fact, if I click into it again, that could be really hard to work with from CLIs or using `curl` or `wget`.

So for my URL, I really want to put in a hyphen.
So let's go ahead and rename our job one more time here.
Let's change this to dash.
I click on rename and now my pipeline is named "hello-world".
If I look up in my URL, it's also a `hello-world`.
That's exactly what I wanted, but really, it's still not what I want.
Now at "hello-world", I'm okay with here, but looking at my dashboard, I see `hello-world`.
Well, really what I want to show, especially when I'm on the dashboard view in this case, is I want to give this a more human-friendly name.
Instead of a URL friendly name, I want to have a human friendly name.

Let's go into our job and click on *Configure* and let's scroll down to *Advanced*.
Now depending on which job that you're using, this next field, which is called *Display Name*, may show up in different places.
Sometimes it will be exposed, sometimes it will be under *Advanced*.
So in this case, what I want to have is I want to say, "Hello World from the universe."
Let's click on *Save*.
We can see here that our Pipeline name now is "Hello World from the universe", but our project name or our job name is still "hello-world".
So if we take a look up in the URL, it's still `hello-world`, but in the breadcrumb, we see the full display name that we just typed in.
In fact, when we go back to the root of our dashboard, we can see "Hello World from the universe".

If you have any questions or comments, you can reach out to us on Twitter @CloudBeast.
If this video was helpful to you, give us a thumbs up.
And if you haven't subscribed to CloudBeesTV yet, why not? Take a moment, click on that subscribe button, and then ring that bell, and you'll be notified anytime there's new content available on CloudBeesTV.
Thanks for watching, and we will see you in the next video.
```